clients/src/main/java/org/apache/kafka/clients/admin/ListShareGroupsResult.java

```diff
@@ -1,84 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License. You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.kafka.clients.admin;
-
-import org.apache.kafka.common.KafkaFuture;
-import org.apache.kafka.common.annotation.InterfaceStability;
-import org.apache.kafka.common.internals.KafkaFutureImpl;
-
-import java.util.ArrayList;
-import java.util.Collection;
-
-/**
- * The result of the {@link Admin#listShareGroups(ListShareGroupsOptions)} call.
- * <p>
- * The API of this class is evolving, see {@link Admin} for details.
- */
-@InterfaceStability.Evolving
-public class ListShareGroupsResult {
```

Comment:
Could you please remove those classes from [KIP-932](https://cwiki.apache.org/confluence/display/KAFKA/KIP-932%3A+Queues+for+Kafka#KIP932:QueuesforKafka-ListShareGroupsResult)

Reply:
Done. I have aligned KIP-932 with KIP-1043. This means:
* Removal of `Admin.listShareGroups` and the associated classes.
* Replacement of `ShareGroupState` with the generic `GroupState`.